### DISCUSSION TITLE
Wrong/misleading release info

### DISCUSSION THREAD

**Discussion by Andre601** [simple-icons/simple-icons#9780] [General]
The latest release (9.19.0) mentions that the jekyll icon has been changed.

However, looking at the linked PR did the icon not receive any changes, making this info a lie.

If there was a change should the release text be updated to point to the right PR/commit updating the icon.
And if there wasn't a change should the entry be removed from the release notes.

Thanks.

**Comment by adamrusted** [comment]
Good spot. I can only assume because the icon _above_ Jekyll received an update - the release action that generated the note picked up Jekyll in error. Will update now (if I can!)

**Reply by Andre601** [reply]
Note that I only checked jekyll. Not sure if there aren't other ones too.

**Reply by adamrusted** [reply]
There was actually another icon that was wrongly labelled (JFrog Bintray) - updated the release note now.
